strings/base_string.h

```diff
@@ -326,6 +326,38 @@ WINRT_EXPORT namespace winrt
         {
             return rend();
         }
+        
+        inline bool starts_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[0] == value;
+        }
+
+        inline bool ends_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[size() - 1] == value;
+        }
+
+#if _HAS_CXX20
```

Comment:
[According to this](https://en.cppreference.com/w/cpp/feature_test), there is a feature testing macro we can use: `__cpp_lib_starts_ends_with`